config/routes.rb

```diff
@@ -1,10 +1,12 @@
 TmcServer::Application.routes.draw do
+  get 'org/:organization_id/course_templates', to: 'course_templates#list_for_teachers', as: 'list_course_templates'
+  get 'org/:organization_id/course_templates/:id', to: 'course_templates#prepare_course', as: 'prepare_course'
```

Comment:
not a big fan of manual matchers; we probably could turn these into members of the ` resources :organizations, path: 'org' do`